### DISCUSSION TITLE
[Issue] nexo print server problem

### DISCUSSION THREAD

**Discussion by lim10soft** [Blair2004/NexoPOS#1599] [General]
In order to seamlessly receive orders from tablets and phones, I need to use the Nexo Print Server as HTTP. HTTPS requires permission, which can be inconvenient. However, when using HTTP, my computer cannot detect the printers with the IP address. It can detect them when using HTTPS. It can be detected from localhost using HTTP.

Interestingly, when using cloud hosting, I encounter this issue. However, when using a local server, this problem doesn't occur.

https://github.com/Blair2004/NexoPOS/assets/102427740/9e424a36-07cd-4972-aae6-80e20d2c98bb

**Comment by Blair2004** [comment]
Hi,
Yes in fact we're really limited with the actual restrictions applied to browsers. To solve this, we have a Windows app that will reintroduce a more faster cloud printing.

This thread will be moved to discussions as the situation is known.
